projects/sunbird-pdf-player/src/lib/sunbird-pdf-player.component.ts

```diff
@@ -52,15 +58,13 @@ export class SunbirdPdfPlayerComponent implements OnInit, OnDestroy, OnChanges,
   }
 
   ngOnInit() {
-    if (this.playerConfig.metadata['compatibilityLevel']) {
-    if (this.playerConfig.metadata.compatibilityLevel > this.defaultCompatibilityLevel) {
-      const  compatibilityError = new Error();
-      compatibilityError.message = `player supports ${this.defaultCompatibilityLevel} and content is compatibility is ${this.playerConfig.metadata.compatibilityLevel}`;
-      compatibilityError.name = 'contentCompatibily';
-
+    const isCompitable = this.contentCompabilityService.
```

Comment:
Update code with below and test
`    const contentCompabilityLevel = this.playerConfig.metadata['compatibilityLevel'];
    if(contentCompabilityLevel) {
      const {isCompitable, error } = this.contentCompabilityService.
      checkContentCompatibility(contentCompabilityLevel);
      if (!isCompitable) {
        this.viewerService.raiseErrorEvent(error, 'compatibility-error');
      }
    }`